client-src/elements/chromedash-stack-rank-page.ts

```diff
@@ -34,22 +35,14 @@ export class ChromedashStackRankPage extends LitElement {
     ];
   }
 
-  static get properties() {
-    return {
-      type: {type: String}, // "css" or "feature"
-      view: {type: String}, // "popularity" or "animated"
-      viewList: {attribute: false},
-      tempList: {attribute: false},
-    };
-  }
-
-  constructor() {
-    super();
-    this.type = '';
-    this.view = '';
-    this.viewList = [];
-    this.tempList = [];
-  }
+  @property({type: String, attribute: false})
+  type!: 'css' | 'feature';
+  @property({type: String, attribute: false})
+  view!: 'popularity' | 'animated';
+  @property({type: Array, attribute: false})
+  viewList: any[] = [];
+  @property({type: Array, attribute: false})
+  tempList: any[] = [];
```

Comment:
Per [@jyasskin's comment](https://github.com/GoogleChrome/chromium-dashboard/pull/4089#discussion_r1672824243), this should likely be changed to having 1 of either `type` or `attribute`.